### PR TITLE
rustify if statements style

### LINKED ISSUE #17699: cleanup `orientation_of_angle` once #17696 lands.
#17696 is introducing a style that is unheard-of in Servo's codebase:

```rust
        if      rounded_angle < 0.25 * PI { Orientation::Angle0   }
        else if rounded_angle < 0.75 * PI { Orientation::Angle90  }
        else if rounded_angle < 1.25 * PI { Orientation::Angle180 }
        else if rounded_angle < 1.75 * PI { Orientation::Angle270 }
        else                              { Orientation::Angle0   }
```

Instead of this, we should probably use a more rusty style, either early-return in multiple lines:

```rust
if rounded_angle < 0.25 * PI {
    return Orientation::Angle0
}

if rounded_angle < 0.75 * PI {
    return Orientation::Angle90
}

// ...
```

Or in one line (not a fan, but in this case it can be argued that it looks nicer):

```rust
if rounded_angle < 0.25 * PI { return Orientation::Angle0 }
if rounded_angle < 0.75 * PI { return Orientation::Angle90 }
// ...
```

### PR DESCRIPTION
**Pull request by SergeevPavel**
Rewrite if statement in more rusty style.

---

- [X] `./mach build -d` does not report any errors
- [X] `./mach test-tidy` does not report any errors
- [X] These changes fix #17699 (github issue number if applicable).

- [ ] There are tests for these changes OR
- [ ] These changes do not require tests because _____

---
This change is [Reviewable](https://reviewable.io/reviews/servo/servo/17745)

### CODE REVIEW

File: components/style/properties/longhand/inherited_box.mako.rs [hunk 1]
```diff
@@ -164,11 +164,19 @@ ${helpers.single_keyword("image-rendering",
...
+        if rounded_angle < 0.75 * PI {
+            return Orientation::Angle90
+        }
+        if rounded_angle < 1.25 * PI {
+            return Orientation::Angle180
+        }
+        if rounded_angle < 1.75 * PI {
+            return Orientation::Angle270
+        }
+        return Orientation::Angle0;
```
Comment (nox) [inline components/style/properties/longhand/inherited_box.mako.rs]: Explicit `return` for terminal expression isn't rustic. :)

Comment (SergeevPavel) [inline components/style/properties/longhand/inherited_box.mako.rs]: Fix it.
I tried to find some code style description, but I could not.

**Comment by emilio** [comment]
Could you squash your commits please? Thanks!

**Comment by emilio** [comment]
@bors-servo r+

Thanks!

**Pull request merged**